pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dlt"
-version = "0.4.1a0"
+version = "0.4.1a1"
```

Comment:
should we add a test somewhere to check that the version number is in sync everywhere?

Reply:
what do you mean. there's just one source of truth for the version and this is the toml file. or you want to have a tests that compares the toml file with the installed package? or maybe that should be a lint step where we force people to `make dev` when versions are not in sync?